charts/backstage/templates/servicemonitor.yaml

```
@@ -0,0 +1,37 @@
+{{- if .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "common.names.fullname" $ }}
+  namespace: {{ .Release.Namespace | quote }}
+  {{- if or .Values.commonAnnotations .Values.metrics.serviceMonitor.annotations  }}
+  annotations:
+    {{- if $.Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.metrics.serviceMonitor.annotations }}
+    {{ toYaml .Values.metrics.serviceMonitor.annotations | nindent 4 }}
```

Let's allow templating here:

```suggestion
    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.serviceMonitor.annotations "context" $ ) | nindent 4 }}
```